dev/release/VERIFY.md

```diff
@@ -74,3 +74,35 @@ You need to set `PKG_CONFIG_PATH` to find libffi on macOS:
 ```console
 % export PKG_CONFIG_PATH=$(brew --prefix libffi)/lib/pkgconfig:$PKG_CONFIG_PATH
 ```
+
+### C++, C#, C GLib, Go, JavaScript, Python, Ruby
+
+Example scripts to install the dependencies to run the verification 
+script for verifying the source on Ubuntu 20.04, Rocky Linux 8 and 
+AlmaLinux 8 are in this folder and named `setup-ubuntu.sh` and 
+`setup-rhel-clone.sh`. These can be adapted to different 
+situations. Go and JavaScript are installed by the verification 
+script in the testing environment. 
+
+Verifying the binaries additionally requires installation of Docker.
+Verifying the wheels additionally requires running
+```console
+git submodule init
+git submodule update
+export PARQUET_TEST_DATA="${PWD}/cpp/submodules/parquet-testing/data"
+export ARROW_TEST_DATA="${PWD}/testing/data"
+```
+from the arrow directory.
```

Comment:
We don't need this because the verification script does this automatically.

Reply:
```
export PARQUET_TEST_DATA="${PWD}/cpp/submodules/parquet-testing/data"
export ARROW_TEST_DATA="${PWD}/testing/data"
```
are in the verification script but
```
git submodule init
git submodule update
```
were not and some of the wheel tests fail without this.